Detectors/TRD/simulation/src/Detector.cxx

```diff
@@ -173,7 +173,13 @@ bool Detector::ProcessHits(FairVolume* v)
     fMC->TrackPosition(xp, yp, zp);
     tof = tof * 1e6; // The time of flight in micro-seconds
     const int trackID = stack->GetCurrentTrackNumber();
-    addHit(xp, yp, zp, tof, totalChargeDep, trackID, det, drRegion);
+    double pos[3] = { xp, yp, zp };
+    double loc[3] = { -99, -99, -99 };
+    gGeoManager->SetCurrentPoint(pos);
+    gGeoManager->FindNode();
```

Comment:
This is not necessary. The gGeoManager already knows the path/node at hit processing. We simply need to transform the global coordinates into local ones.

Reply:
Thanks for pointing this out. I didn't know that it was unnecessary.